src/node.cc

```diff
@@ -981,6 +983,42 @@ void SetupNextTick(const FunctionCallbackInfo<Value>& args) {
       FIXED_ONE_BYTE_STRING(args.GetIsolate(), "_setupNextTick"));
 }
 
+void PromiseRejectCallback(PromiseRejectMessage message) {
+  Isolate* isolate = Isolate::GetCurrent();
```

Comment:
Scratch this for now. I know there was a time when it was going to be deprecated (I know this because I spent half a day working around it myself at one point), but I can't verify that right now.